homeassistant/components/mqtt/client.py

```diff
@@ -393,7 +399,6 @@ def __init__(
         self._wildcard_subscriptions: list[Subscription] = []
         self.connected = False
         self._ha_started = asyncio.Event()
-        self._last_subscribe = time.time()
```

Comment:
When this is removed, I think there will be a crash if there are no subscriptions, no discovery messages and a birth message is configured because `self._last_subscribe` is not defined?
Also, do we need to reset `self._last_subscribe` when reconnecting to the broker?

Reply:
Is it not that only during `self._discovery_cooldown` this is used? An there it is initialize too.